docker/Dockerfile

```diff
@@ -16,7 +18,7 @@ RUN	apt update \
 	&& curl -fsSl https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - \
 	&& add-apt-repository "deb [arch=amd64] https://apt.llvm.org/xenial/ llvm-toolchain-xenial-12 main" \
 	&& apt update \
-	&& apt -y install clang-12 clang-tools-12 lld-12 build-essential pkg-config autoconf git python docker-ce-cli pciutils gcc-multilib libgcc-8-dev-arm64-cross
+	&& apt -y install clang-12 clang-tools-12 lld-12 build-essential pkg-config autoconf git python docker-ce-cli pciutils gcc-multilib libgcc-8-dev-arm64-cross gcc-mingw-w64-x86-64
```

Comment:
would this help with docker images on windows based containers?

Reply:
I tried messing around with Windows containers a while back and found them nearly impossible to use — incredibly slow and things froze up all the time. But maybe!